doc/5/core-classes/kuzzle/query/index.md

```diff
@@ -42,7 +42,7 @@ Refer to Kuzzle's API Reference [here](/core/1/api)
 
 ## query
 
-## `query` is a JSON object containing arguments specific to the query, such as a `body` property, a JWT hash, a document `_id`, or generic query options (such as `from` or `size` for [search queries](/core/1/api/controllers/document/search))
+## `query` is a JSON object containing arguments specific to the query, such as a `body` property, a JWT hash, a document `_id`, or generic query options (such as `from` or `size` for [search queries](/core/1/api/controllers/collection/search))
```

Comment:
Does not exist either.

Should probably be:

```suggestion
## `query` is a JSON object containing arguments specific to the query, such as a `body` property, a JWT hash, a document `_id`, or generic query options (such as `from` or `size` for [search queries](/sdk/js/5/core-classes/collection/search))
```

Reply:
Seems weird because I've done this change to fix a 404 on [this page](https://docs.kuzzle.io/sdk/js/5/core-classes/kuzzle/query/) on the "search queries" link but the [linked page](https://docs.kuzzle.io/core/1/api/controllers/document/search) exists and the `/core/1/api/controllers/document/search` link to 404.
Looks like the link was ending by `/` because it add the baseUrl of JS5 sdk and redirect on `sdk/js/5/core/1/api/controllers/document/search/` 
Can't remember why a change `document` to `collection`.

Reply:
document.search does not exist in the javascript sdk 5